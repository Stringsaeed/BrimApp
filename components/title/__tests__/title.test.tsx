import React from "react";
import { render, screen } from "@testing-library/react-native";

import Title from "..";

describe("Title", () => {
  it("should render correctly", () => {
    render(<Title>Test</Title>);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
