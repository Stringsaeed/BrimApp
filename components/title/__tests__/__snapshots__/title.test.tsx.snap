// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<Text
  style={
    {
      "fontSize": 36,
      "fontWeight": "700",
    }
  }
>
  Test
</Text>
`;
